packages/browser-audio-input/src/pcm-recorder.ts

```diff
@@ -85,6 +101,34 @@ export class PCMRecorder extends TypedEventTarget<PCMRecorderEventMap> {
     this.dispatchTypedEvent(RECORDING_STARTED, new Event(RECORDING_STARTED));
   }
 
+  mute() {
+    if (!this.mediaStream) return;
+
+    for (const track of this.mediaStream.getTracks()) {
+      console.log(track);
+      track.enabled = false;
+    }
+
+    this.dispatchTypedEvent(MUTE, new MuteEvent());
+  }
+
+  unmute() {
+    if (!this.mediaStream) return;
+
+    for (const track of this.mediaStream.getTracks()) {
+      track.enabled = true;
+    }
+
+    this.dispatchTypedEvent(UNMUTE, new UnmuteEvent());
+  }
+
+  get muted() {
```

Comment:
Would it be better to name is `isMuted` as it returns a boolean?

Reply:
Yeah that's what I called it in the React hook too, so should probably change it here for consistency.

At first I was thinking of mirroring the existing Web API names (i.e. `track.enabled`), but `isMuted` is probably better for this.